sdl_android_lib/src/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -783,6 +785,11 @@ public void onCreate() {
 			stopSelf();
```

Comment:
This section is getting a bit large and should be moved to a single function with a name like initCheck() that returns a boolean if the checks passed.